.github/config/evergreen-arm-hardfp.json

```diff
@@ -15,6 +15,7 @@
     "evergreen-arm-hardfp",
     "evergreen-arm-hardfp-sbversion-15",
     "evergreen-arm-hardfp-sbversion-16"
```

Comment:
Missed a trailing comma here

Reply:
I powered through creating this CL, fixed now! Ty for catching that.